cmd/errors.go

```diff
@@ -151,3 +151,8 @@ func isErr(err error, errs ...error) bool {
 	}
 	return false
 }
+
+// Errorf behaves like fmt.Errorf but also traces the returned error.
+func Errorf(format string, args ...interface{}) error {
```

Comment:
Actually the reason is that traceErrorf() indicates that you are taking in formatting arguments as well and also along with errors.